### DISCUSSION TITLE
CentOS support

### DISCUSSION THREAD

**Discussion by aleonccc** [deployphp/deployer#3712] [Help needed]
Hi there!

I wonder if CentOS is supported by Deployer. 

My VPS runs on CentOS.

Thanks.

**Comment by antonmedv** [comment]
Yes, it should run on a centOS.

**Comment by aleonccc** [comment]
Sorry, my fault:
I meant to deploy a project on a CentOS.
Sorry again.

**Reply by antonmedv** [reply]
Yes. It should
